packages/app-cli/tests/support/test_notes/directory/non-empty/non-empty/sample.md

```diff
@@ -0,0 +1,3 @@
+# Markdown
+
+Test sample.
```

Comment:
Please remove all these test files. Usually when we have such files it's to test things like parsing and rendering, but in this case it doesn't make much sense. Could you create them dynamically in a temp directory and delete them when it's done? Use createTempDir from test-utils.ts for this.

Reply:
Done